docs/guides/other-guides/forwarding-and-load-balancing-with-cloud-endpoints.mdx

```diff
@@ -148,46 +61,33 @@ and most common use cases: forwarding traffic to other endpoints
 	}}
 />
 
-## **Step 4** — Create a ngrok Cloud Endpoint
+## **Step 4** — Start an agent session
 
-Now we can create a ngrok Cloud Endpoint that points to our
-newly created internal endpoint. Cloud Endpoints can be created
-via the API or ngrok dashboard.
+At this point, you have a public endpoint with a domain and a traffic policy routing to the URL we defined in our traffic policy above, `https://the-internal-endpoint.internal`. But that URL doesn't exist on the public internet; we need to start an agent session so that requests to our public endpoint will be routed to our internal endpoint, served by the ngrok agent. So, locally, start an agent session with:
```

Comment:
```suggestion
Finally, to accept requests routed to your internal endpoint, you must start an ngrok agent locally on the machine or resource you want to use with the following command.
```
I heavily simplified this, but let me know if I've made it inaccurate. I appreciate that the wording was deliberate to make what's happening clearer, but I personally found it harder to process than a more streamlined explanation